cmd/virt-launcher/node-labeller/node-labeller.sh

```diff
@@ -32,6 +32,10 @@ if [ -e /dev/sev ]; then
   chmod o+rw /dev/sev
 fi
 
+if [ -e /var/lib/kubevirt-node-labeller/setcap-dac-override-enable ]; then
```

Comment:
In the context of the handler, this is fine even without the `if` imho. But I wonder why you don't get this capability.
Did you investigate this issue? I would expect this will cause issues for you elsewhere as well.

Reply:
Maybe libvirt isn't built with the macro `WITH_CAPNG` enabled on alpine?

Reply:
@xpivarc well, @zedi-pramodh had spent lots of time with @vasiliy-ul, with issue reported in issue  #8615 , and others have seen similar issue, but so far no other way around.